README.md

```diff
@@ -1 +1,16 @@
-# rails-forms
\ No newline at end of file
+# README
```

Comment:
Let's be a little more descriptive in here. What's the project about? What did you learn? How can I get setup? Do I need to run `bundle install`, `rails db:setup` and `rails s`? And after I run the server, do I simply navigate to `localhost` or should I go to `/users/new` to be able to create a user? And to edit a user what should I do (is it `/users/1/edit`)?

[`REQUIRED`]